lib/util/usedPropTypes.js

```diff
@@ -314,6 +364,7 @@ module.exports = function usedPropTypesInstructions(context, components, utils)
           const propName = ast.getKeyValue(context, properties[k]);
 
           if (propName) {
+            propVariables.set(propName, parentNames.concat([propName]));
```

Comment:
```suggestion
            propVariables.set(propName, parentNames.concat(propName));
```
(unless `propName` could be an array)